api/capi/capi-nntrainer.pc.in

```diff
@@ -8,5 +8,6 @@ Name: tizen-api-nntrainer
 Description: NNTrainer API for Tizen
 Version: @VERSION@
 Requires:
+# Requires: capi-ml-common
```

Comment:
is this intended?

Reply:
Yes, it should be changed once the package is ready. will be changed to line:11 as PRs in nnstreamer side gets merged.